Gemfile

```diff
@@ -6,6 +6,7 @@ gem 'jquery-rails', '~> 4.3'
 #gem 'twitter-bootstrap-rails', :github => 'seyhunak/twitter-bootstrap-rails', :ref => 'c236286b7d6e73affa1597f62fd51e9dbd268f1c'
 gem 'twitter-bootstrap-rails'
 gem 'sprockets-rails', '~> 3.2'
+gem 'mustache-js-rails'
```

Comment:
@wwjbatista Actually, one comment: could you specify a version? That way, we don't have unexpected surprises when running `bundle update`...